Cargo.toml

```diff
@@ -75,6 +76,14 @@ tokio-stream = { workspace = true, features = ["net"] }
 message-hub = ["astarte-device-sdk/message-hub"]
 systemd = ["dep:systemd"]
 forwarder = ["dep:edgehog-forwarder"]
+udev = ["dep:udev"]
+wifiscanner = ["dep:wifiscanner"]
+rauc = ["dep:zbus"]
+upower = ["dep:zbus"]
+cellular = ["dep:zbus"]
+battery = ["dep:zbus"]
+led = ["dep:zbus"]
+dbus_hid = ["dep:zbus"]
```

Comment:
mmm, I don't know if we want to specify all as features, perhaps we can use directly zbus as discriminant.
For example enable led_behavior only checking zbus.
``` rust
#[cfg(feature = "zbus")]
Led(led_behavior::LedEvent),
```
What do you think?

Reply:
Yes, it makes sense, in your opinion which features should be enabled by default by zbus ?
Should I merge them all ? I would keep the `dbus_hid` separated since if disabled it requires the user to set a device id

Reply:
Since we have to keep retro-compatibility all these zbus features should be enabled by default.
I think also `dbus_hid` can be merged, an user can pass a device_ID via config as other configurations. 
I think we can start simple keep all below `dep:zbus`.

Reply:
We could also add a cfg!(not([target_os](https://doc.rust-lang.org/reference/conditional-compilation.html#target_os) = "linux")) to keep it as default but permit the build on other platforms